docs/docs/integrations/datadog.mdx

```diff
@@ -1,100 +1,65 @@
-GrowthBook with [DataDog RUM](https://www.datadoghq.com/product/real-user-monitoring/).
+### Setup
 
-To get started, you'll first need to instrument DataDog's RUM with their beta [Feature Flag Tracking](https://docs.datadoghq.com/real_user_monitoring/feature_flag_tracking/) feature.
+:::tip Info
 
-When initializing `datadogRum` you can add `enableExperimentalFeatures` and pass in `feature_flags` to get access to the beta feature.
+If your DataDog RUM browser SDK version is below v5.17.0, [enable feature flags during RUM SDK initialization](https://docs.datadoghq.com/real_user_monitoring/guide/setup-feature-flag-data-collection/?tab=browser#setup).
 
-```ts
-datadogRum.init({
-  applicationId: "sample-application-id",
-  clientToken: "sample-client-token",
-  site: "datadoghq.com",
-  service: "demo-app",
-  env: "dev",
-  version: "1.0.0",
-  sessionSampleRate: 100,
-  sessionReplaySampleRate: 100,
-  trackUserInteractions: true,
-  trackResources: true,
-  trackLongTasks: true,
-  defaultPrivacyLevel: "mask-user-input",
-// This enables the beta feature flag tracking
-  enableExperimentalFeatures: ["feature_flags"],
-});
-
-datadogRum.startSessionReplayRecording();
-```
-
-Then, you can use the `addFeatureFlagEvaluation` method built in to the `datadogRum` package when evaluating GrowthBook Feature Flags.
-
-```ts
-import { datadogRum } from "@datadog/browser-rum";
-import { useFeatureIsOn } from "@growthbook/growthbook-react";
+:::
 
-const welcomeMessage = useFeatureIsOn("welcome-message");
+We can report all feature flag evaluations to DataDog using GrowthBook’s SDK `trackingCallback` function, as shown in the snippet below.
 
-datadogRum.addFeatureFlagEvaluation("welcome-message", welcomeMessage);
+```
+trackingCallback: (experiment, result) => {
+  datadogRum.addFeatureFlagEvaluation(experiment.key, result.key);
+}
 ```
 
-:::info Note
-
-If you are using [DataDog APM](https://www.datadoghq.com/product/apm/), you can do something similar by adding Feature Flag data to your traces. DataDog then
-has a helpful guide on how to [Connect Rum & Traces](https://docs.datadoghq.com/real_user_monitoring/connect_rum_and_traces/?tab=browserrum).
-This route will enable you do have alerts built around both front-end and back-end errors.
-
-:::
-
-### 2. Build a DataDog Dashboard & Monitor
+### Advanced example
 
 Now that you&apos;re passing Feature Flag data to DataDog, you can build monitors and alerts when the error rate between Feature Flags variations cross a specific threshold within a pre-defined time period.
 
-![](../../static/images/integrations/datadog/add-feature-flags-to-dashboard-data-dog.png)
+#### 1. Create a DataDog monitor
 
 The beta Feature Flag Tracking feature allows you to see error rates between the various feature flag states.
 
 ![](../../static/images/integrations/datadog/view-feature-flags-error-rates-in-data-dog.png)
```

Comment:
Since we're updating, could you add descriptive alt text, please. Helps with SEO/a11y